modules/rosa-hcp-aws-privatelink-security-groups.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+[id="rosa-hcp-aws-privatelink-security-groups_{context}"]
+:_mod-docs-content-type: PROCEDURE
+= Configuring AWS security groups to access the API
+
+With {hcp-title} private clusters, the AWS PrivateLink endpoint exposed in the customer's VPC has a default security group. This security group has access to the PrivateLink endpoint that is limited to only those resources that exist within the VPC or resources that are present with an IP address associated with the VPC CIDR range. In order to grant access to any entities outside of the VPC, through VPC peering and transit gateway, you must create and attach another security group to the PrivateLink endpoint to grant the necessary access.
+
+.Prerequisites
+
+* Your corporate network or other VPC has connectivity.
+* You have permission to create and attach security groups within the VPC.
+
+.Procedure
+
+. Set your cluster name as an environmental variable by running the following command:
++
+[source,terminal]
+----
+$ export CLUSTER_NAME=<cluster_name>
+----
++
+You can verify that the variable has been set by running the following command:
++
+[source,terminal]
+----
+$ echo $CLUSTER_NAME
+----
++
+.Example output
++
+[source,terminal]
+----
+hcp-private
+----
+
+. Use the following command to find the VPC endpoint (VPCE) ID and VPC ID:
```

Comment:
Could be flipped around to match the more common "by running the following command" style:
```suggestion
. Find the VPC endpoint (VPCE) ID and VPC ID by running the following command:
```